packages/tooltip/src/tooltipUtils.tsx

```diff
@@ -0,0 +1,116 @@
+import { css } from '@leafygreen-ui/emotion';
+
+export function trianglePosition(
+  alignment: string,
+  justification: string,
+  triggerRect: DOMRect | ClientRect | null,
+) {
+  if (!alignment || !justification || !triggerRect) {
+    return '';
+  }
+
+  const borderOffset = 'calc(100% - 1px)';
+
+  const containerSize = 15;
+  const notchSize = 8;
+  const containerOffsetX = (triggerRect.width - containerSize) / 2;
+  const containerOffsetY = (triggerRect.height - containerSize) / 2;
+  const notchOverlap = -notchSize / 2;
+  const containerStyleObj: {
+    left?: string;
+    right?: string;
+    top?: string;
+    bottom?: string;
+    margin?: string;
+  } = {};
```

Comment:
There's gotta be a better way to type this...

Reply:
👍